BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+
+**M. Àngels** -:  Next meeting is going to be next week, but it won't match Reza's schedule.
+
+**Matthias** -: Yeah. Reza is in Canada so it's also in this time zone.
+
+**M. Àngels** -:  So shall we put this on hold until Reza can jump into the meeting in three weeks?
+
+**Matthias** -: We can definitely continue the conversations on the side as it has been happening, but not on Discord channel, if possible. And get it back on, I guess, the CIP meeting, again in the US based time zone.
+
+**M. Àngels** -:  Okay.
+
+**Matthias** -: We can also have a quick catch-up on it next week just to see where we're at again.
+
+**M. Àngels** -:  Okay. And the thing that I would like to clarify because it is not clear to me is if we are going to do on specific channel on Discord or for the moment, it's still better to try to use GitHub as a more public reflected and official channel.
+
+**Matthias** -: I would prefer GitHub. If people strongly want a Discord channel for discussing this one, I think we can also go for that.
+
+**Robert** -: I think the only way we're going to see the improvement is if we create the Discord channel, I can't think of a better test case we will get for a topic specific discussions than this CIP we have right now. And at this panel, within a month or two, we could have similarly contended proposals. And we don't want anyone to have to do the work that Matthias just did in order to digest four weeks of discussions. So we can in the hopes of getting through it all on a single meeting.
+
+**Robert** -: So can we try to separate them and see if it works? See if we can rely upon the people to keep bringing it back into GitHub once some consensus or important points are made on Discord. And then if it seems like the balance is coming out wrong, if it seems like things are only being recorded in Discord, but not being taken back to GitHub, then we can maybe look at how that how we could make that process work. I just [crosstalk 00:36:49].
+
+**M. Àngels** -:  Can you explain it again, at least from my end, there was a little bit of background noise and I couldn't understand. I mean, I think that you were suggesting to keep the discussion on Discord, but make everyone owner of trespassing their comments.
+
+**Robert** -: I mean, a lot of the Discord on Discord. A lot of the discussion on Discord was repetitive and we don't want that happening on GitHub. A lot of the repetitive discussion on Discord was opinionated. And we need a chance for the opinions to play out before they're recorded as actual point of view to be recorded in GitHub. So without having a forum for people to get through the preliminaries, we end up either recording too much on GitHub, which is the opinions and the bickering, or we end up recording too little because people forget to bring back the main point from the social networking channel to the developer's record on GitHub.
```

Comment:
```suggestion
**Robert** -: I mean, a lot of the "discord" on Discord. A lot of the discussion on Discord was repetitive and we don't want that happening on GitHub. A lot of the repetitive discussion on Discord was opinionated. And we need a chance for the opinions to play out before they're recorded as actual point of view to be recorded in GitHub. So without having a forum for people to get through the preliminaries, we end up either recording too much on GitHub, which is the opinions and the bickering, or we end up recording too little because people forget to bring back the main point from the social networking channel to the developer's record on GitHub.
```